### PR TITLE
docs: add storybook V5 configuration for typescript project

### PR DESCRIPTION
**Pull request by ipatate**
i have added the rule for use storybook with typescript.

### CODE REVIEW

File: docs/docs/visual-testing-with-storybook.md [hunk 1]
```diff
@@ -99,6 +99,29 @@ module.exports = ({ config }) => {
 
 > Note that if you're using a [StaticQuery](/docs/static-query/) in your components, `babel-plugin-remove-graphql-queries` is required to render them in Storybook. This is because the queries are run at build time in Gatsby, and will not have been run when rendering the components directly.
 
+> For typescript projet, add this rule:
```
Comment (wardpeet) [inline docs/docs/visual-testing-with-storybook.md]: Unsure if this is better phrasing.
```suggestion
> When using typescript, add this rule:
```

File: docs/docs/visual-testing-with-storybook.md [hunk 2]
```diff
@@ -99,6 +99,29 @@ module.exports = ({ config }) => {
...
+```js:title=.storybook/webpack.config.js
+...
+
+config.module.rules.push({
+    test: /\.(ts|tsx)$/,
+    loader: require.resolve('babel-loader'),
+    options: {
+      presets: [['react-app', {flow: false, typescript: true}]],
+      plugins: [
+        require.resolve('@babel/plugin-proposal-class-properties'),
```
Comment (wardpeet) [inline docs/docs/visual-testing-with-storybook.md]: Just wondering if we need `@babel/plugin-proposal-class-properties`. Doesn't typescript has class properties support by default?

File: docs/docs/visual-testing-with-storybook.md [hunk 3]
```diff
@@ -99,6 +99,29 @@ module.exports = ({ config }) => {
 
 > Note that if you're using a [StaticQuery](/docs/static-query/) in your components, `babel-plugin-remove-graphql-queries` is required to render them in Storybook. This is because the queries are run at build time in Gatsby, and will not have been run when rendering the components directly.
 
+> For typescript projet, add this rule:
+
+```js:title=.storybook/webpack.config.js
+...
```
Comment (wardpeet) [inline docs/docs/visual-testing-with-storybook.md]: I think this is a typo, it messes up the markdown output.
```suggestion
```

File: docs/docs/visual-testing-with-storybook.md (hunk 2 above)
Comment (ipatate) [reply docs/docs/visual-testing-with-storybook.md]: i tried without this plugins and i have a error "Support for the experimental syntax 'classProperties' isn't currently enabled".
It's required.

Comment (wardpeet) [reply docs/docs/visual-testing-with-storybook.md]: 👍  thanks for validating.

**Pull request merged**